errors/no-img-element.md

```diff
@@ -1,51 +1,59 @@
-# No Img Element
+# No img element
 
-> Prevent usage of `<img>` element to prevent layout shift.
+> Prevent usage of `<img>` element to prevent layout shift and favor [optimized images](https://nextjs.org/docs/basic-features/image-optimization).
 
 ### Why This Error Occurred
 
-An `<img>` element was used to display an image. Use either `<picture>` in conjunction with `<img>` element, or use `next/image` that has better performance and automatic Image Optimization over `<img>`.
+An `<img>` element was used to display an image.
 
 ### Possible Ways to Fix It
 
-Import and use the `<Image />` component:
+Use [`next/image`](https://nextjs.org/docs/api-reference/next/image) to improve performance with automatic [Image Optimization](https://nextjs.org/docs/basic-features/image-optimization).
+
+> Note: If deploying to a [managed hosting provider](https://nextjs.org/docs/deployment), remember to check pricing since optimized images might be charged differently that the original images. If self-hosting, remember to install [`sharp`](https://www.npmjs.com/package/sharp) and check if your server has enough storage to cache the optimized images.
 
 ```jsx
 import Image from 'next/image'
 
 function Home() {
   return (
-    <>
-      <Image
-        src="https://example.com/test"
-        alt="Landscape picture"
-        width={500}
-        height={500}
-      />
-    </>
+    <Image
+      src="https://example.com/hero.jpg"
+      alt="Landscape picture"
+      width={800}
+      height={500}
+    />
   )
 }
 
 export default Home
```

Comment:
Is there a reason we didn't include the section on `unoptimized`?

Reply:
Its on line 32 but linked to instead of copy pasta. This doc is for lint, not really for `next/image` itself.